packages/authorization-process/src/routers/AuthorizationRouter.ts

```diff
@@ -106,8 +106,50 @@ const authorizationRouter = (
       authorizationMiddleware([ADMIN_ROLE]),
       async (_req, res) => res.status(501).send()
     )
-    .get("/clients", authorizationMiddleware([ADMIN_ROLE]), async (_req, res) =>
-      res.status(501).send()
+    .get(
+      "/clients",
+      authorizationMiddleware([
+        ADMIN_ROLE,
+        SECURITY_ROLE,
+        M2M_ROLE,
+        SUPPORT_ROLE,
+      ]),
+      async (req, res) => {
+        const ctx = fromAppContext(req.ctx);
+        try {
+          const { name, userIds, consumerId, purposeId, kind, offset, limit } =
+            req.query;
+          const clients = await authorizationService.getClients(
+            {
+              name,
+              userIds: userIds?.map(unsafeBrandId<UserId>),
+              consumerId: unsafeBrandId(consumerId),
+              purposeId: purposeId
+                ? unsafeBrandId<PurposeId>(purposeId)
+                : undefined,
+              kind,
+            },
+            { offset, limit },
+            req.ctx.authData,
+            ctx.logger
+          );
+          return res
+            .status(200)
+            .json({
+              results: clients.results.map((client) =>
+                clientToApiClient(client, {
+                  includeKeys: false,
+                  showUsers: ctx.authData.organizationId === client.consumerId,
+                })
+              ),
+              totalCount: clients.totalCount,
+            })
+            .end();
+        } catch (error) {
+          const errorRes = makeApiProblem(error, () => 500, ctx.logger);
```

Comment:
In the other PR we are still creating the error mapper even if there only error 500, can you do the same here?

Reply:
i think it's the best way if there are no mistakes other than 500

Reply:
I agree with @Carminepo2 , it creates a bit of overhead but it allows to use a standardized approach

Reply:
Ok, then I'll do the same for `createConsumerClient` and  `createApiClient` as well